config/datanode.example.toml

```diff
@@ -101,3 +101,17 @@ parallel_scan_channel_size = 32
 # [logging]
 # dir = "/tmp/greptimedb/logs"
 # level = "info"
+
+# datanode export the metrics generated by itself
+# encoded to Prometheus remote-write format
+# and send to Prometheus remote-write compatible receiver (e.g. send to `greptimedb` itself)
+# This is only used for `greptimedb` to export its own metric internally. Please see `logging` option for normal export of metric.
+# [system_metric]
+# whether enable export system_metric, default is false
+# enable = false
+# The url of metric export endpoint, default is `greptimedb` default frontend endpoint
+# endpoint = "127.0.0.1:4000"
+# The database name of exported metrics stores, user needs to specify a valid database
+# db = ""
```

Comment:
let's default to `information_schema`

Reply:
the `information_schema` in the db is a virtual table, user still need create `information_schema` manually in db, then user can export metrics. the final result would be something like this, which is very confusing
![image](https://github.com/GreptimeTeam/greptimedb/assets/46434815/2cfa5cff-a7c9-4cd6-9fc0-fceae9c24134)

Reply:
leave empty first, wait https://github.com/GreptimeTeam/greptimedb/issues/2964 solve then change the default value